pc/nzp/default.cfg

```diff
@@ -87,6 +87,7 @@ name "Unknown Soldier"
 // CONTROLLER BINDS
 //
 seta cl_controllerglyphs "xbox"
+seta in_rumbleenable 1
```

Comment:
The cvar you made was `in_rumbleenabled`, with a `d` at the end. This lacks that. Can you fix?